modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/main_view/info_panel/components/AssetTags.tsx

```diff
@@ -130,7 +130,7 @@ const AssetTags = ({
 							onClick: createAndAddKeyword,
 						}
 					}
-					refetchOnActive
+					refetchOnActive={true}
```

Comment:
Since `refetchOnActive` is a boolean property when it's passed with no value it's evaluated as `true`, so we can remove `={true}`.